src/components/TrackingTrigger/TrackingTrigger.js

```diff
@@ -25,24 +25,24 @@ class TrackingTrigger extends PureComponent {
     static propTypes = {
         /** The event to trigger. */
         event: PropTypes.string.isRequired,
-        /** The event specific fields. */
-        fields: PropTypes.objectOf(PropTypes.string),
+        /** The event specific payload. */
```

Comment:
For backwards compatibility, this still needs to also accept `fields` property but in `componentDidMount` the `trigger` should allow `payload` to take precedence if specified.